src/chatdlg.cpp

```diff
@@ -98,7 +98,7 @@ void CChatDlg::AddChatText ( QString strChatText )
 #if QT_VERSION < QT_VERSION_CHECK(5, 0, 0)
         txvChatWindow, 0, QAccessible::ValueChanged
 #else
-        new QAccessibleEvent ( txvChatWindow, QAccessible::ValueChanged )
+        new QAccessibleValueChangeEvent ( txvChatWindow, strChatText )
```

Comment:
I just noticed that you changed the QAccessibleEvent. Can you please explain what was wrong with the previous version? Did it not work as expected?

Reply:
When I built and ran the code in Qt 5.9, using the Debug configuration, this line would raise an assertion in [qaccessible.h](https://github.com/qt/qtbase/blob/dev/src/gui/accessible/qaccessible.h#L675), so I followed the instructions in the Qt code and used the recommended QAccessibleEvent 
 subclass.

Reply:
Ah, ok. Thanks for the info. BTW: I merged your code. Thank you for your contribution.